### PR TITLE
St/schedule data

### PR DESCRIPTION
**Pull request by showduhtung**

### CODE REVIEW

File: scripts/makeDatabase.js [hunk 1]
```diff
@@ -1,20 +1,26 @@
+/* eslint-disable array-callback-return */
```
Comment (showduhtung) [inline scripts/makeDatabase.js:1]: for the most part we can skip this file as having perfect code in this file is not necessary. it's just a few scripts I wrote up to help me generate large files to seed the database.

File: scripts/seed.js [hunk 2]
```diff
@@ -18,34 +20,26 @@ async function seed() {
...
-    await new Promise((r) => setTimeout(r, 2000));
+    const seedChurches = await db.Church.bulkCreate(churches);
+    const seedSchedules = await db.Schedule.bulkCreate(schedules);
+    const seedDividers = await db.Divider.bulkCreate(dividers);
 
-    const seedUsers = await Promise.all(users.map((user) => db.User.create(user)));
-    await new Promise((r) => setTimeout(r, 2000));
-
-    const seedRoles = await Promise.all(roles.map((role) => db.Role.create(role)));
-    await new Promise((r) => setTimeout(r, 2000));
```
Comment (hchiam) [inline scripts/seed.js:28]: nice! just using sequelize bulkCreate

File: shared/SequelizeTypings/models/EventModel.ts [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
+import { CommonSequelizeAttributes } from '.';
 
-export interface EventAttributes {
-    id?: number;
-    time: Date;
-    tag: string;
+export interface EventAttributes extends CommonSequelizeAttributes {
+    roleId?: number; // team
+    day: string;
+    order: number;
```
Comment (hchiam) [inline shared/SequelizeTypings/models/EventModel.ts:7]: i'm assuming `order` is just a number to indicate which events happen earlier relative to other events? sounds like an alternative proxy for time/date

File: shared/SequelizeTypings/models/RequestModel.ts [hunk 4]
```diff
@@ -14,8 +14,6 @@ export interface RequestAttributes {
     replace?: boolean;
```
Comment (hchiam) [inline shared/SequelizeTypings/models/RequestModel.ts:14]: Nice! So someone can ask to give you their task, or assign(?) a task to you, not just swap tasks with you.

File: scripts/seed.js (hunk 2 above)
Comment (showduhtung) [reply scripts/seed.js:28]: right, I totally forgot this existed oops. the previous implementation was creating instances out of order and that was a problem

File: shared/SequelizeTypings/models/EventModel.ts (hunk 3 above)
Comment (showduhtung) [reply shared/SequelizeTypings/models/EventModel.ts:7]: not exactly. it's an indicator for how it should be ordered on the schedule. for example

10:45 AM    ||   Hymn Leading
                    ||   Piano
                    ||   Usher

On the schedule, you can tell that Hymn leading, piano and usher are all at 10:45AM, but not all of them will display their time on the row. You also don't want your rows to be out of order accidentally.

File: shared/SequelizeTypings/models/RequestModel.ts (hunk 4 above)
Comment (showduhtung) [reply shared/SequelizeTypings/models/RequestModel.ts:14]: I think so, right @alan-mj-lin ?

**Review by hchiam — APPROVED** [review]
LGTM! :+1:

File: shared/SequelizeTypings/models/RequestModel.ts (hunk 4 above)
Comment (alan-mj-lin) [reply shared/SequelizeTypings/models/RequestModel.ts:14]: I believe that is the intention yes

**Review by hchiam — APPROVED** [review]

**Review by hchiam — APPROVED** [review]
LGTM! :+1:

**Pull request merged**